src/TestFramework/Coverage/CoverageMethodData.php

```diff
@@ -50,21 +50,9 @@ final class CoverageMethodData
      */
     public $endLine;
 
-    /**
-     * @var int
-     */
-    public $executed;
-
-    /**
-     * @var int
-     */
-    public $coverage;
-
-    public function __construct(int $startLine, int $endLine, int $executed, int $coverage)
+    public function __construct(int $startLine, int $endLine)
```

Comment:
This method no longer contains information about the coverage data of the method, just about where it is located in the file, so it probably needs renaming. I can't find a good name for it atm.

Reply:
`LineCodeCoverage`?